Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -453,8 +454,9 @@ changeDriverEnableState admin personId isEnabled = do
     QPerson.findById personId
       >>= fromMaybeM (PersonDoesNotExist personId.getId)
   unless (person.merchantId == admin.merchantId) $ throwError Unauthorized
-  QDriverInformation.updateEnabledState driverId isEnabled
-  unless isEnabled $ QDriverInformation.updateActivity driverId False
+  Esq.runTransactionF $ \finalize -> do
+    CQDriverInformation.updateEnabledState finalize driverId isEnabled
+    unless isEnabled $ CQDriverInformation.updateActivity (Esq.ignoreFinalize finalize) driverId False
```

Comment:
Why finalize has to be ignored here? Anyway it would be better if you just pass `pure ()` here/ create `CQDriverInformation.updateActivity` version without finalize requirement. So `ignoreFinalize` is not required IMO

Reply:
> Why finalize has to be ignored here?
Its ignored because we don't need to clear the same cache twice.

> Anyway it would be better if you just pass `pure ()` here

Not just `pure ()`, it should be like `const @(m()) (pure ())`, and `m` should be the same as external monad, so I guess `Esq.ignoreFinalize` is more clear. Anyway I can remove, its not a big deal

Reply:
Okay, leave it be. Can you remove finalize as argument in it then? it's not used anyway

Reply:
Of course I can add second version `updateActivity'` without finalize. But its not very good, because everyone will use it and cache will be not cleared again. In my solution cache will be cleared by default, but if you don't want it, then you should say it explicitly